README.md

```diff
@@ -239,6 +277,8 @@ Variables should be named as descriptively as possible. Single letter variable n
 
 Asterisks indicating pointers belong with the variable, e.g., `NSString *text` not `NSString* text` or `NSString * text`, except in the case of constants.
 
+Collection variables should be templatized when possible to make explicit the types of the contents of the collection.
+
```

Comment:
A short example would improve the statement.

Reply:
There was an example below in the code block but I added one inline to make more clear what the statement is talking bout 👍